src/frontend/Sider/Personoversikt/Aktivitetsoversikt/Aktiviteter.tsx

```diff
@@ -0,0 +1,58 @@
+import React from 'react';
+
+import { styled } from 'styled-components';
+
+import { Table } from '@navikt/ds-react';
+
+import { Aktivitet, statutAktivitetTilTekst } from './typer';
+import { formaterNullableIsoDato } from '../../../utils/dato';
+import { formatBoolean } from '../../../utils/tekstformatering';
+
+const Aktivitetstabell = styled(Table)`
```

Comment:
```suggestion
const Tabell = styled(Table)`
```
Så ser det kanskje litt mindre ut som en egen komponent?